lib/data_hygiene/publishing_api_html_attachment_redirector.rb

```diff
@@ -37,7 +39,11 @@ def unpublished_or_withdrawn
     end
 
     def html_attachments
-      @html_attachments ||= last_edition.html_attachments
+      @html_attachments ||= document ? last_edition.html_attachments : [html_attachment]
+    end
+
+    def html_attachment
+      @html_attachment ||= HtmlAttachment.find_by(content_id: content_id)
```

Comment:
It seems a bit awkward to me to try and either use the content_id to find the document, or the HTML attachment.

Does this worker change the state of the HTML attachment in the Whitehall database?

Reply:
No it doesn't change the state.  It just calls the Publishing API.  I started writing a separate rake task for individual HTML Attachments but the code was so similar to this that it made sense to me to just include it here.  Is there a particular reason why we shouldn't use the content id to find an HTML Attachment?  We do have precedent in the app for checking if a content id is for a Document or HTML Attachment https://github.com/alphagov/whitehall/blob/master/app/controllers/admin/documents_controller.rb#L2

Reply:
Only that it seems awkward to use the content_id to either find a document, or a HTML attachment.

Saying that, I was thinking this was part of the app, but it's actually just used in a Rake task right? In that case, I don't think it matters as much.